pkg/agent/openflow/pipeline.go

```diff
@@ -2858,6 +2858,7 @@ func (f *featurePodConnectivity) hostBridgeLocalFlows() []binding.Flow {
 		ClassifierTable.ofTable.BuildFlow(priorityNormal).
 			Cookie(cookieID).
 			MatchInPort(f.uplinkPort).
+			MatchDstMAC(f.nodeConfig.UplinkNetConfig.MAC).
```

Comment:
Please check if should support multicast/broadcast MAC.

Reply:
I check the multicast changes for IPAM Pods, the added flows are with higher property in ClassifierTable matching L3 multicast CIDR. But I don't test with broadcast. May I know the sceraio for such traffic??

Reply:
Broadcast MAC is used for broadcast traffic, such as ARP req.

Reply:
ClassifierTable is to process IP packets, ARP packets is handled in ARP pipeline. So do you know any required cases with IP broadcast?

Reply:
I think the most useful broadcast traffic is ARP/ICMPv6, but since we passes all traffic to host network before, this change may break worker node to receive broadcast traffic, unless we restrict user to use broadcast in worker node with bridgingMode.